src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -1399,6 +1399,9 @@
   <data name="Argument_NoUninitializedStrings" xml:space="preserve">
     <value>Uninitialized Strings cannot be created.</value>
   </data>
+  <data name="Argument_NoUninitializedUtf8Strings" xml:space="preserve">
+    <value>Uninitialized Utf8Strings cannot be created.</value>
```

Comment:
Maybe just change `Strings` to `strings` in the other message so that it applies to all strings? I do not think that the specialized message is worth it for this very specialized API - if we want to make this better we should include the actual type name in all errors coming from this API.